packages/metal-dom/src/dom.js

```diff
@@ -254,6 +254,31 @@ class dom {
 	}
 
 	/**
+	 * Verifies if the element is able to trigger the Click event,
+	 * simualting browsers behaviour, avoiding event listeners to be called by dom.triggerEvent method.
+	 * @param {Element} node Element to be checked.
+	 * @param {!String} eventName The event name.
+	 */
+	static isAbleToInteract_(node, eventName) {
+		var currElement = node;
+		var isAble = true;
+		var matchesSelector = 'button, input, select, textarea, fieldset';
+
+		if (eventName === 'click') {
+			while (currElement && dom.match(currElement, matchesSelector) && !dom.match(currElement, 'legend')) {
```

Comment:
In this case I'm testing if a parent of the node passed as argument is a legend tag. Because, according to W3C, if a 'button, input, select or textarea' are children of a disabled 'fieldset' they should be disabled, unless, those children are wrapped by a legend.